sdk/communication/azure-communication-networktraversal/src/main/java/com/azure/communication/networktraversal/implementation/models/CommunicationIceServerRouteType.java

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.communication.networktraversal.implementation.models;
+
+import com.azure.core.util.ExpandableStringEnum;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import java.util.Collection;
+
+/** Defines values for CommunicationIceServerRouteType. */
+public final class CommunicationIceServerRouteType extends ExpandableStringEnum<CommunicationIceServerRouteType> {
```

Comment:
Is Ice a well-known acronym for users to understand the difference between this enum and the RequestRouteType? The javadoc should be improved too.